devops/providers/azure-devops/templates/infrastructure/scripts/set-cicd-flag.sh

```diff
@@ -9,6 +9,12 @@ function setCICDFlag() {
     echo "##vso[task.setvariable variable=needs_cicd;isOutput=true]true"
 }
 
+if [ ! -z ${FORCE_CICD} ]; then
```

Comment:
Can you sync with @awkwardindustries to align on an approach on this? There is a different implementation in her work here: https://github.com/microsoft/cobalt/pull/280/files#diff-8aba13cd2c66fc5e866ee42469d0a725R29